apps/sertra/components/mod-selector/search.tsx

```diff
@@ -8,6 +7,7 @@ interface SearchBoxProps<T> {
   keyExtractor: (option: T) => string | number;
   options: T[];
   onSelect?: (option: T) => void;
+  searchVars: string[];
```

Comment:
This now allows us to pass any string in `searchVars` prop, including ones that are not actually keys of type `T`. Which means that `option[searchVar]` below will return undefined, and calling `.toLowerCase()` breaks the execution. At least for me the IDE actually warns about `option[searchVar]`, but with the state of linting/configs in this repo YMMV.

Anyways, to fix the issue, use `searchVars: (keyof T)[];` instead. TypeScript will now ensure that the parameters passed are actually keys used in `T`. In practice this means that if there's e.g. a typo at `modal.txs`'s `searchVars={["ownerName", "packageName"]}`, IDE will warn about it and the build process will fail. This re-generalizes the component (referring to your first commit message here).

Also IMO `searchKeys` might be a better name, since the valuea aren't really variable.